### DISCUSSION TITLE
[Feature request] Show thumbnail for .exe files in Deepin file manager

### DISCUSSION THREAD

**Discussion by SiamSami** [linuxdeepin/developer-center#3019] [Features Request & Ideas | 特性请求 & 头脑风暴]
As the title says, it would be very good if Deepin file manager showed thumbnails for .exe files. Even dolphin and nemo has this feature.
We can use [exe-thumbnailer](https://github.com/exe-thumbnailer/exe-thumbnailer) for this.

**Comment by SiamSami** [comment]
@BLumia Is it going to be implemented?

**Comment by meiyixiang** [comment]
Thank you for your advice

**Comment by meiyixiang** [comment]
We have adopted this suggestion and scheduled to implement it, but it may take some time due to the large demand

**Comment by yaaaaa** [comment]
Have anyone any solution?

**Reply by BLumia** [reply]
It's marked as suggestion get accepted so it need some time for development.
